server/api/v1.py

```diff
@@ -564,6 +564,29 @@ def rate_review_for_user():
 
     return vote_added_response
 
+@api.route('/user/remove_scholarship_from_profile', methods=['PUT'])
```

Comment:
Good call, I like the idea of using the DELETE method instead of PUT. Seems kind of odd though to use the logged in user for the request, (not quite stateless), i.e. RETful rather than RESTUL.